openassessment/xblock/static/js/src/lms/oa_response.js

```diff
@@ -707,10 +721,13 @@ OpenAssessment.ResponseView.prototype = {
             }
         });
 
-        $(this.element).find('.file__upload').prop('disabled', isError);
+        this.uploadEnabled(!isError);
```

Comment:
- Please update the comment to reflect the return value.
- Now that the function returns a value, 'checkFilesDescriptions' name seems a bit odd. Perhaps 'filesDescriptionsValid' would be a better name?

Reply:
In addition to checking validity, it also has the side-effect of enabling the upload button if the descriptions are valid and storing the file description values in `this.filesDescriptions`.  `checkFileDescriptions` and `fileDescriptionsValid` basically connote the same idea ("I'm going to return false if something is wrong with the descriptions").  Maybe `collectFileDescriptions`?  When it returns `true`, they will have been collected, and when it returns `false`, nothing will have been collected.